iglovikov_helper_functions/data_processing/gleason2019/prepare_the_data.py

```diff
@@ -111,22 +114,39 @@ def get_mapping() -> np.array:
     return mapping
 
 
-def main():
-    args = parse_args()
+def merge_masks(file_path: str):
+    mask_list = []
+    file_path = Path(file_path)
+    final_mask_path = file_path.parents[1] / "train" / "masks" / (file_path.stem + ".png")
+    if not final_mask_path.exists():
+        for num_expert in range(6):
```

Comment:
I think we should be more explicit here:

`range(1, 6 + 1)`

and avoid `num_expert + 1` later